### PR TITLE
Vault Updates

### PR DESCRIPTION
**Pull request by pruett**
🔍 [Preview](https://deploy-preview-1--vault-updates.netlify.com/)

- Expose /docs and /intro views
- Use documentation-style layout for index pages
- Editable subnav

------

Additional Views:

- [x] #2 [Use Case] Secrets Management
- [x] #3 [Use Case] Data Encryption
- [x] #4 [Use Case] Identity Based Access

### CODE REVIEW

**Comment by pruett** [comment]
@mwickett Here's a quick and dirty preview of updated homepage. Note. There are a few places where I'm referencing external data via a call to dato like so: `dato.vault_product_page.static_dynamic_section.layout.content`. Let me know if you think this should be brought locally in the project. (Some of the data was fairly large like for the logo grids).

Preview: https://5daf65f46b9d6d000af5a0a9--vault-updates.netlify.com/

**Comment by pruett** [comment]
Still spacing and aesthetic issues, wanted to get eyes on this in early stages...will be working on use case pages...

**Comment by mwickett** [comment]
This is looking great, thanks Kevin. I think given the constraints imposed here (our components expecting Dato data, needing the `encode` step etc.) the way you've done it here makes sense. 

I'll make sure to let Chris Kent know exactly which areas are coupled with .com (logo grids)

**Comment by pruett** [comment]
@mwickett Quick update here:

- Homepage feedback addressed: https://deploy-preview-1--vault-updates.netlify.com/
- New use case route added: https://deploy-preview-1--vault-updates.netlify.com/use-cases/secrets-management/

**Comment by kylemac** [comment]
@pruett lets improve the spacing on that first section. i think you just need to override the padding-top on 0px. 

Screen Shot 2019-10-23 at 4 58 36 PM

**Comment by pruett** [comment]
- [ ] improve the spacing on first section (https://github.com/pruett/vault/pull/1#issuecomment-545684579)

**Comment by pruett** [comment]
Update:

- `/use-cases/data-encryption` route added: https://deploy-preview-1--vault-updates.netlify.com/use-cases/data-encryption/

**Comment by pruett** [comment]
Update:

- `/use-cases/identity-based-access` route added: https://deploy-preview-1--vault-updates.netlify.com/use-cases/identity-based-access/

File: website/source/netlify-redirects [hunk 1]
```diff
@@ -169,16 +175,16 @@
...
+# /intro/getting-started/dev-server.html https://learn.hashicorp.com/vault/getting-started/dev-server
+# /intro/getting-started/first-secret.html https://learn.hashicorp.com/vault/getting-started/first-secret
+# /intro/getting-started/secrets-engines.html https://learn.hashicorp.com/vault/getting-started/secrets-engines
+# /intro/getting-started/dynamic-secrets.html https://learn.hashicorp.com/vault/getting-started/dynamic-secrets
+# /intro/getting-started/help.html https://learn.hashicorp.com/vault/getting-started/help
+# /intro/getting-started/authentication.html https://learn.hashicorp.com/vault/getting-started/authentication
+# /intro/getting-started/policies.html https://learn.hashicorp.com/vault/getting-started/policies
+# /intro/getting-started/deploy.html https://learn.hashicorp.com/vault/getting-started/deploy
+# /intro/getting-started/apis.html https://learn.hashicorp.com/vault/getting-started/apis
+# /intro/getting-started/next-steps.html https://learn.hashicorp.com/vault/getting-started/next-steps
```
Comment (pruett) [inline website/source/netlify-redirects]: I'm assuming I can remove all of these `/intro` => `learn.hashicorp` redirects as this update is now embracing the `/intro` route?

cc/ @mwickett @kylemac

Comment (mwickett) [reply website/source/netlify-redirects]: I think I missed that this shift with Intro was happening. I'm going to confirm with Chris Kent about this.

**Comment by mwickett** [comment]
Did we stop getting preview builds here? I still see `inner.erb` as a an `<h1>` on `/intro`

**Comment by pruett** [comment]
@mwickett Yeah, let me check to make sure the builds are kicking off properly...

**Pull request merged**